static/bidder-params/taboola.json

```diff
@@ -12,7 +12,12 @@
       "type": "string"
     },
     "tagid": {
-      "type": "string"
+      "type": "string",
+      "description": "Deprecated, use tagId instead."
+    },
+    "tagId": {
+      "type": "string",
+      "description": "preferred"
```

Comment:
Could we mention here in the description as - `preferred, will get precedence if both tagId and tagid are defined` ?
The reason is there is no way to tell that tagId gets precedence without looking into the code. It would be good if we capture this information here in the json schema.

Reply:
yes - makes sense. 
Added - thank you :)